phpBB/styles/prosilver/theme/bidi.css

```diff
@@ -502,10 +502,17 @@ li.breadcrumbs span:first-child > a {
 
 .rtl blockquote cite {
 	/* Username/source of quoter */
-	margin-right: 20px;
+	margin-right: 0;
 	margin-left: 0;
 }
 
+.rtl blockquote cite:before, .uncited:before {
+	/* Font Awesome quote-right */
+	content: '\f10e';
+	padding-right: 0;
+	padding-left: 5px;
+}
+
```

Comment:
might make more sense that this be separated out and moved to the icon.css file as well as long as it follows the code for the left
```css
.rtl blockquote cite:before,
.uncited:before {
	content: "\f10e"; /* Font Awesome quote-right */
}
```